### PR TITLE
Gateway host env

### PR DESCRIPTION
**Pull request by rem1niscence**
move gateway host string to an environment variable

### CODE REVIEW

File: tests/acceptance/v1.controller.acceptance.ts [hunk 1]
```diff
@@ -1983,8 +1983,7 @@ describe('V1 controller (acceptance)', () => {
       expect(response.body.error.message).to.startWith('Restricted endpoint: contract address not allowed')
     })
 
-    // eslint-disable-next-line mocha/no-exclusive-tests
-    it.only('invokes POST /v1/{appId} and successfully relays a request only through the altruist', async () => {
+    it('invokes POST /v1/{appId} and successfully relays a request only through the altruist', async () => {
```
Comment (adshmh) [inline tests/acceptance/v1.controller.acceptance.ts:1991]: This doesn't related to this PR, could you clarify why it is needed?

**Review by adshmh — CHANGES_REQUESTED** [review]
Could we add tests to cover both valid and invalid sub-domain sceanrios?

**Review by crisog — DISMISSED** [review]
Looks good, straight forward 👍🏽 

If #977 gets merged, this ENV won't be needed in the future so I'll add a note in there.

**Review by adshmh — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**